src/.nuget/optdata/ibcmerge.csproj

```diff
@@ -23,10 +23,14 @@
     <!-- Error if IbcMergePackagePathOutputFile is not set. -->
     <Error Condition="'$(IbcMergePackagePathOutputFile)'==''" Text="IbcMergePackagePathOutputFile must be passed as a property." />
 
+    <PropertyGroup>
+      <IbcMergePath>$(NuGetPackageRoot)microsoft.dotnet.ibcmerge/$(MicrosoftDOtNetIBCMergeVersion)</IbcMergePath>
```

Comment:
Non-blocking nit - casing (DOtNet) - no big deal as msbuild treats properties as case insensitive.

Reply:
NuGet casing is lowercase for restored packages.